### ISSUE TITLE
[site:mode] prod command should disable cacheability headers

### ISSUE THREAD

**Issue by drubb** [hechoendrupal/DrupalConsole#2322]
When using the site:mode prod command, the following services.yml is created:

```
parameters:
    session.storage.options: { gc_probability: 1, gc_divisor: 100, gc_maxlifetime: 200000, cookie_lifetime: 2000000 }
    twig.config: { debug: false, auto_reload: false, cache: true }
    renderer.config: { required_cache_contexts: ['languages:language_interface', theme, user.permissions], auto_placeholder_conditions: { max-age: 0, contexts: [session, user], tags: {  } } }
    http.response.debug_cacheability_headers: true
    factory.keyvalue: {  }
    factory.keyvalue.expirable: {  }
    filter_protocols: [http, https, ftp, news, nntp, tel, telnet, mailto, irc, ssh, sftp, webcal, rtsp]
```

As you can see, the cacheability headers are enabled, I think they should be disabled (false).

**Comment by enzolutions** [comment]
Hi @drubb 

Thank you for your proposal. But after checking the issue https://www.drupal.org/node/2527126, looks like this setting if not related with dev mode or prod. 

This option is more related with the kind of CND or reverse proxy are you using.

Please let me know if you are agree and we could close this issue, or maybe provide more support for your proposal.

**Comment by drubb** [comment]
Hi @enzolutions,

the issue on d.o. is very long, but basically it says two things:

- We'd prefer to disable those headers, because the create very long headers and impose a security risk, telling to much details about the site
- In case the cacheability headers are used  by a cdn or reverse proxy for invalidation, we can't disable them.

I'd consider the second one a special case. Normally, no cdn or reverse proxy knows anything about Drupal cache stuff, they need to be configured to use them, in which case you could explicitely enable the headers in production.

Maybe take a look at default.services.yml, suggesting to disable the headers in production:
```
  # Cacheability debugging:
  #
  # Responses with cacheability metadata (CacheableResponseInterface instances)
  # get X-Drupal-Cache-Tags and X-Drupal-Cache-Contexts headers.
  #
  # For more information about debugging cacheable responses, see
  # https://www.drupal.org/developing/api/8/response/cacheable-response-interface
  #
  # Not recommended in production environments
  # @default false
  http.response.debug_cacheability_headers: false
```
So I still think those headers were added for debugging purposes (and then abused for invalidating assets), and should be disabled in production by default.

**Comment by enzolutions** [comment]
Hi @drubb 

Thank you for your clarification, I just add a PR https://github.com/hechoendrupal/DrupalConsole/pull/2968 to resolve your feature request and will be included in 1.0.0-rc10
